backends/dpdk/dpdkProgramStructure.h

```diff
@@ -39,6 +39,9 @@ struct DpdkProgramStructure {
     std::map<const IR::Declaration_Instance *, cstring>         csum_map;
     std::map<cstring, int>                                      error_map;
     std::vector<const IR::Declaration_Instance *>               externDecls;
+    std::map<cstring, std::vector<cstring>>                     key_map;
+    std::map<cstring, const IR::P4Table *> group_tables;
```

Comment:
why not indent these the same as the previous ones?

Reply:
Ack. Will fix it in the next commit.